tests/ui/suggestions/dont_suggest_raw_pointer_syntax-issue-127562.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0594]: cannot assign to `*ptr`, which is behind a `*const` pointer
+  --> $DIR/dont_suggest_raw_pointer_syntax-issue-127562.rs:5:9
+   |
+LL |         *ptr = 3;
+   |         ^^^^^^^^ `ptr` is a `*const` pointer, so the data it refers to cannot be written
+   |
+help: consider specifying this binding's type
```

Comment:
This diagnostic comes out only in UI test, but will not come out when run in command line, this is because of https://github.com/rust-lang/rust/issues/127562#issuecomment-2226784298

Reply:
originally because span location here is remapped: 
https://github.com/chenyukang/rust/blob/b44a48416d06524da2ea247f1d1973d85ef514f8/compiler/rustc_borrowck/src/diagnostics/mutability_errors.rs#L1133

`$SRC_DIR/core/src/ptr/mod.rs:LL:COL (#4), location: bb0[4]`